packages/now-cli/src/util/dev/builder-cache.ts

```diff
@@ -180,7 +180,7 @@ export async function installBuilders(
 
   // Filter out any packages that come packaged with `now-cli`
   const packagesToInstall = packages.filter(p =>
-    filterPackage(p, distTag, buildersPkgBefore)
+    filterPackage(p, distTag, buildersPkgBefore, nowCliPkg)
```

Comment:
I thought I brought this up in the previous PR and you said it was supposed to use the global scope 🤔

Reply:
I mentioned that "all uses in now dev pass in the global one", but it was fake news, because it was missed in this case 🤦